### PR TITLE
Adding geoffreys sculptures

### PR DESCRIPTION
**Pull request by nejurgis**
What this MR does:

- Adds Sebs/Geoffreys sculpture V1
- Tweaks eslint/prettier

### CODE REVIEW

File: src/templates/Studio/Studio.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ export default ({ location, data: { indexPage, studioPage } }) => {
   return (
     <Layout>
       <SEO title="Studio" pathName={studioPage.fields.slug} />
-      <Wonder introRef={introRef} />
+      <Wonder />
```
Comment (larixk) [inline src/templates/Studio/Studio.js]: Can the `introRef` variable be completely removed? Does not seem like it's doing anything.

File: src/templates/Studio/Wonder/Mirrors/Mirrors.js [hunk 2]
```diff
@@ -17,10 +17,14 @@ const Mirrors = ({ layout, scene, world }) => {
     const activeMirrorContent = [];
     const addMirrors = () => {
       layout.forEach(mirrorLayout => {
-        const mirror = MeshBuilder.CreatePlane('glass', {
-          width: mirrorLayout.width,
-          height: mirrorLayout.height,
-        }, scene);
+        const mirror = MeshBuilder.CreatePlane(
```
Comment (larixk) [inline src/templates/Studio/Wonder/Mirrors/Mirrors.js]: Please remove unused files (Mirrors.js, and any other parts of previous scene).

Removing unused code keeps the repo cleaner and we will still be able to find them in the Git history when we need them.

File: src/templates/Studio/Wonder/Sun/Sun.js [hunk 3]
```diff
@@ -1,9 +1,5 @@
-import React, { useEffect } from 'react';
-import {
-  Animation,
-  DirectionalLight,
-  Color3,
-} from 'babylonjs';
+import { useEffect } from 'react';
```
Comment (larixk) [inline src/templates/Studio/Wonder/Sun/Sun.js]: Is the lighting of the sculpture still based on the position of the sun in Amsterdam or is this legacy code that's not really used anymore?

File: src/templates/Studio/Wonder/World/World.js [hunk 4]
```diff
@@ -9,10 +9,31 @@ const World = ({ filename, layout, onImportWorld, scene }) => {
...
+
+      const material = new StandardMaterial('CustomSTANDARTMaterial', scene);
+      if (belt) {
+        belt.material = material;
+      }
+
+      // makes belt lighter
+      material.emissiveColor = new Color3(1, 1, 1);
+
+      const texture = new Texture('models/sculpture/checkers_big.png');
```
Comment (larixk) [inline src/templates/Studio/Wonder/World/World.js]: Having trouble finding this file. Did you add it?

Comment (nejurgis) [reply src/templates/Studio/Wonder/World/World.js]: Yeah I deleted it by mistake previously but re-added it

File: src/templates/Studio/Wonder/Sun/Sun.js (hunk 3 above)
Comment (nejurgis) [reply src/templates/Studio/Wonder/Sun/Sun.js]: Still using that beautiful code written by Andy, couldn't lift my hand to delete it

File: src/templates/Studio/Wonder/Mirrors/Mirrors.js (hunk 2 above)
Comment (nejurgis) [reply src/templates/Studio/Wonder/Mirrors/Mirrors.js]: Removed

File: src/templates/Studio/Studio.js (hunk 1 above)
Comment (nejurgis) [reply src/templates/Studio/Studio.js]: Removed

**Pull request merged**